docs/reference/glossary.rst

```diff
@@ -0,0 +1,62 @@
+Glossary
+========
+
+There are a variety of terms used in the documentation and throughout the
+library. Here is an overview:
+
+An action
+---------
+
+An **action** is either :code:`C` or :code:`D`.
+You can access these actions as follows but should not really have a reason to::
+
+    >>> import axelrod as axl
+    >>> axl.Actions.C
+    'C'
+    >>> axl.Actions.D
+    'D'
+
+A play
+------
+
+A **play** is a single player choosing an **action**.
+In terms of code this is equivalent to::
+
+    >>> p1, p2 = axl.Cooperator(), axl.Defector()
+    >>> p1.play(p2)  # This constitues two 'plays' (p1 plays and p2 plays).
+
+This is equivalent to :code:`p2.play(p1)`. Either function invokes both
+:code:`p1.strategy(p2)` and :code:`p2.strategy(p1)`.
+
+A turn
+------
+
+A **turn** is a 1 shot interaction between two players. It is in effect a
+composition of two **plays**.
+
+Each turn has four possible outcomes of a play: :code:`(C, C)`, :code:`(C, D)`,
+:code:`(D, C)`, or :code:`(D, D)`.
+
+A match
+-------
+
+A **match** is a consecutive number of **turns**. The default number of turns
+used in the tournament is 200. Here is a single match between two players over
+10 turns::
+
+    >>> p1, p2 = axl.Cooperator(), axl.Defector()
+    >>> for turn in range(10):
+    ...     p1.play(p2)
+    >>> p1.history, p2.history
+    (['C', 'C', 'C', 'C', 'C', 'C', 'C', 'C', 'C', 'C'], ['D', 'D', 'D', 'D', 'D', 'D', 'D', 'D', 'D', 'D'])
+
+A round robin
+-------------
+
+A **round robin** is the set of all potential (order invariant) matches between
+a given collection of players.
+
```

Comment:
That's a good point. Will do.

On 22 October 2015 at 17:35, Owen Campbell <notifications@github.com> wrote:

> In docs/reference/glossary.rst
> <https://github.com/Axelrod-Python/Axelrod/pull/372#discussion_r42772158>:
>
> > +A **match** is a consecutive number of **turns**. The default number of turns
> > +used in the tournament is 200. Here is a single match between two players over
> > +10 turns::
> > +
> > +    >>> p1, p2 = axl.Cooperator(), axl.Defector()
> > +    >>> for turn in range(10):
> > +    ...     p1.play(p2)
> > +    >>> p1.history, p2.history
> > +    (['C', 'C', 'C', 'C', 'C', 'C', 'C', 'C', 'C', 'C'], ['D', 'D', 'D', 'D', 'D', 'D', 'D', 'D', 'D', 'D'])
> > +
> > +A round robin
> > +-------------
> > +
> > +A **round robin** is the set of all potential (order invariant) matches between
> > +a given collection of players.
> > +
>
> We should mention here that a round robin also includes self-interactions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/372/files#r42772158>.
>



-- 
Dr Vincent Knight
Cardiff School of Mathematics
Senghennydd Road,
Cardiff
CF24 4AG
(+44) 29 2087 5548